README.md

```diff
@@ -1,22 +1,30 @@
 
-# IBM Health Check Operator
+# ibm-healthcheck-operator
 
-## Overview
+Operator used to manage the IBM system healthcheck service.
 
-IBM Health Check Operator is used to manage the IBM health check service
+## Supported platforms
+
```

Comment:
You can install the ibm-healthcheck-operator on Linux® x86_64 with Red Hat® OpenShift® Container Platform version 4.2 or 4.3.